cirq-web/cirq_ts/dist/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <title>Cirq Web Development page</title>
+</head>
+<body>
+  <div id="container"></div>
+  <script src="bloch_sphere.bundle.js"></script>
+  <script>
+    createSphere.showSphere('{"radius": 5}');
```

Comment:
Ah forgot about this..in the `webpack-config-js` file we have the output library name set to "createSphere", but we probably should set it to something more general like even "cirq_web" or something like that. I'll also need to change the showSphere to showBlochSphere or something similar. 

I'll do this after I finish all the other comments since it requires rebundling and changing HTML both on the Python and TS side.